tests/Command/CommandTest.php

```diff
@@ -99,4 +111,28 @@ public function setting_unknown_defaults_throws_an_exception()
             'doesnotexist' => '0',
         ]);
     }
+
+    /**
+     * @test
+     */
+    public function arguments_that_do_not_map_to_defaults_are_not_added()
+    {
+        $this->application->command('greet [name]', [new GreetCommand, 'greet']);
+    }
+
+    /**
+     * @test
+     */
+    public function a_command_with_an_invalid_static_callable_should_not_throw_deprecation_notices()
+    {
+        $this->application->command('greet [name]', [GreetCommand::class, 'greet']);
+    }
```

Comment:
I'm afraid this is invalid: this is not a valid callable, it shouldn't throw deprecation notices but it should throw an exception (see #29)